routes/send.js

```diff
@@ -0,0 +1,46 @@
+var express = require('express');
+var router = express.Router();
+var nodemailer = require('nodemailer');
+
+var transporter = nodemailer.createTransport({
+    service: 'Gmail',
+    auth: {
+        user: 'molecularplayground@gmail.com',
+        pass: 'ihearttim497'
+    }
+});
+
+// setup e-mail data with unicode symbols
+var mailOptions = {
+    from: 'Molecular Playground <molecularplayground@gmail.com>', // sender address
+    to: 'molecularplayground@gmail.com', // list of receivers
+    subject: 'Welcome to Molecular Playground!', // Subject line
+    text: "This is an autogenerated email from Molecular Playground.\nThe body of your message has been lost in the interwebs, sorry!", // plaintext body
+    html: "<p>This is an autogenerated email from Molecular Playground.</p><p>The body of your message has been lost in the interwebs, sorry!</p>" // html body
+};
+
+
+// PUT: send the email!
+router.put('/', function(req, res, next) {
```

Comment:
Just spit-balling here: do you think it would be better to have this section be only for sending generic emails, with the body options email, text, and html?

We could also have a "to-html" function that can be used if specific html is not supplied.